man/spec_delim.Rd

```diff
@@ -47,12 +47,10 @@ spec_csv(
   comment = "",
   trim_ws = TRUE,
   skip = 0,
-  n_max = 0,
   guess_max = 1000,
   name_repair = "unique",
   num_threads = readr_threads(),
   progress = show_progress(),
-  show_col_types = should_show_types(),
   skip_empty_rows = TRUE,
```

Comment:
Since `n_max` will always `= guess_max` I've removed it from the function arguments. I also removed `show_col_types()` because otherwise the column specification would print twice.

Reply:
When I first looked at `spec()` because of this issue, I was sort of surprised at how many arguments there are.

So, yes, I would definitely take a hard look at whether all of them are actually being used / make sense. OTOH, especially, since readr 2e is going to basically parse the file to get the spec, it does makes sense that almost all of the usual arguments are here.